themes/bootstrap3/js/lightbox.js

```diff
@@ -162,7 +162,7 @@ VuFind.register('lightbox', function Lightbox() {
             originalUrl: _originalUrl,
             formUrl: obj.url
           });
-          if (_originalUrl.match(/UserLogin/) || obj.url.match(/catalogLogin/)) {
+          if (_originalUrl.match(/UserLogin/) || obj.url.match(/catalogLogin/) || obj.url.match(/AlmaDatabase/)) {
```

Comment:
Is this match necessary? Is there ever a URL with /AlmaDatabase/ in it?

Reply:
I added that url match because after creating a new account (pushing the "Send" button), the modal dialog did not close itself but issued a page forwarding to the "provide catalog credentials". But that page forwarding had quite a broken layout and was also not necessary. That forwarding only stoped when I added the url match. After you commented on this, I reset the code to before that commit and tried it again so that I could make a screenshot. But now it worked without that page forwarding. I can't explain why (maybe some browser cache issue?!). Anyway: I removed the url match now.